cli/pom.xml

```diff
@@ -87,6 +94,39 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>3.4.0</version>
+        <executions>
+          <execution>
+            <id>attach-artifacts</id>
+            <phase>package</phase>
+            <goals>
+              <goal>attach-artifact</goal>
+            </goals>
+            <configuration>
+              <artifacts>
+                <artifact>
+                  <file>cli-${version}-Linux.tar.gz</file>
```

Comment:
Shouldn't we use `${project.artifactId}` instead hard-coding `cli`?
Also we should align the OS with the syntax we are using in [ide-urls](https://github.com/devonfw/ide-urls/tree/master/java/java/17.0.8_7) and also in custom software repositories what is:
* `linux`
* `mac`
* `windows`

Reply:
I created the tar.gz in the workflow and used the `${{ runner.os }}` for the name, which capitalises `Windows`

done

Reply:
Thanks. Great work 👍 However, you are IMHO still using `macos` instead of `mac`.

Reply:
done